internal/printer/printer_test.go

```diff
@@ -1194,6 +1194,13 @@ const items = ["Dog", "Cat", "Platipus"];
 				code: `<html><body>${$$renderComponent($$result,'Component',Component,{},{"default": () => $$render` + BACKTICK + `<form><textarea></textarea></form>` + BACKTICK + `,})}</body></html>`,
 			},
 		},
+		{
+			name:   "slot inside of Base",
+			source: `<Base title="Home"><div>Hello</div></Base>`,
+			want: want{
+				code: `${$$renderComponent($$result,'Base',Base,{"title":"Home"},{"default": () => $$render` + BACKTICK + `<div>Hello</div>` + BACKTICK + `,})}`,
```

Comment:
Prior to this change, `Base` could not accept any slots because it was being closed as if it were the void tag `<base>`.